api/utils/condition.mapper.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * I don't know if we should convert this to a "sails helper"
+ */
+// const reduce = require('lodash.reduce')
+
+const conditionReducer = (conditions) => {
```

Comment:
Suggestion: Add a quick bit of code comments / documentation to this file to explain what the structure of the `conditions` parameter looks like when it is passed in and what something looks like with it leaves `conditionReducer`.

Reply:
Added some comments. This was a bit of code copied straight over from previous - think it's Calvin's ... think I got the gist of what it's doing. May be worth revisiting this later.